### PR TITLE
Update package template

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

**Description:**

Update the latest package template.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,12 +34,12 @@
     "@dojo/loader": ">=2.0.0-beta.9",
     "glob": "^7.0.3",
```
Comment (kitsonk) [inline package.json]: since we are in here, `grunt-dojo2` includes this, so it is a redundancy

File: package.json [hunk 2]
```diff
@@ -34,12 +34,12 @@
...
     "grunt": "~1.0.1",
-    "grunt-dojo2": ">=2.0.0-beta.26",
+    "grunt-dojo2": ">=2.0.0-beta.34",
     "grunt-text-replace": "^0.4.0",
     "intern": "^3.4.1",
     "istanbul": "^0.4.3",
```
Comment (kitsonk) [inline package.json]: same as above

File: package.json [hunk 3]
```diff
@@ -34,12 +34,12 @@
...
     "remap-istanbul": "^0.6.4",
```
Comment (kitsonk) [inline package.json]: same as above...

File: package.json [hunk 4]
```diff
@@ -34,12 +34,12 @@
     "@dojo/loader": ">=2.0.0-beta.9",
     "glob": "^7.0.3",
     "grunt": "~1.0.1",
-    "grunt-dojo2": ">=2.0.0-beta.26",
+    "grunt-dojo2": ">=2.0.0-beta.34",
     "grunt-text-replace": "^0.4.0",
```
Comment (kitsonk) [inline package.json]: same as above

**Comment by agubler** [comment]
@kitsonk good shout, updated removing those dependencies.

**Pull request merged**